sys/bitfield/bitfield.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_util
+ * @{
+ *
+ * @file
+ * @brief       Bitfield auxillary functions
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include "bitfield.h"
+#include "irq.h"
+
+int bf_get_unset(uint8_t field[], int size)
+{
+    unsigned state = disableIRQ();
```

Comment:
Hm, I actually don't like to introduce more and more sections with disabled interrupts. Think about real-time. 

But, if you think it is inevitable to allow to use this function from ISR context, you could at least move down the instruction as much as possible. I think for sections with disabled interrupts every saved instruction helps.